doc/raspberrypi3.md

```diff
@@ -39,6 +39,16 @@ $ bitbake core-image-minimal
 
 This document uses parted to create partitions and /dev/sde as sdcard device.
 
+## Using SD card image
```

Comment:
Since we have .wic image, it is okay to remove manual install steps(clone firmwares, setup boot.scr, and etc).

Reply:
I left it because there were two procedures described on other boards (ek874 etc.).
I'd like to keep it as another method, but how about it?

Reply:
That's okay. btw,  I recommend change heading levels to separate build steps. 

current heading level is here.

```
## Create sdcard image
## Write kernel image and rootfs manually
## Copy Raspberry Pi firmwares
## Copy kernel, dtb and u-boot
##  Create config.txt
## Setup u-boot
## Setup root file system
```

after
```
## Create sdcard image
## Write kernel image and rootfs manually
### Copy Raspberry Pi firmwares
### Copy kernel, dtb and u-boot
### Create config.txt
### Setup u-boot
### Setup root file system
```